superset/dashboards/api.py

```diff
@@ -752,12 +752,11 @@ def export(self, **kwargs: Any) -> Response:
                 except DashboardNotFoundError:
                     return self.response_404()
             buf.seek(0)
-
             response = send_file(
                 buf,
                 mimetype="application/zip",
                 as_attachment=True,
-                attachment_filename=filename,
+                download_name=filename,
```

Comment:
https://flask.palletsprojects.com/en/2.0.x/api/#flask.send_file
`attachment_filename` is deprecated